lint_models.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint Models
+
+on: [pull_request]
+
+jobs:
+  lint-models:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/setup-python@v2"
+        with:
+            python-version: "3.8"
+      - name: Install SQLFluff
+        run: "pip install sqlfluff==0.3.6"
```

Comment:
I do not know if the quotes break anything but I also do not think they are needed! Maybe we oughtta add some generic SQL to this to ensure that the action runs each time we update it?

Reply:
If folks want to use the dbt templater they would need to run `pip install sqlfluff[dbt]`

So maybe we could create a base github action and then a dbt templater one?

Reply:
The quotes aren't needed, it's valid YAML without them too ;)